src/libraries/System.Private.Uri/tests/FunctionalTests/UriBuilderParameterTest.cs

```diff
@@ -15,9 +15,9 @@ public class UriBuilderParameterTest
         public void UriBuilder_Ctor_NullParameter_ThrowsArgumentException()
         {
             Assert.Throws<ArgumentNullException>(() =>
-           {
-               UriBuilder builder = new UriBuilder((Uri)null);
-           });
+            {
+                _ = new UriBuilder((Uri)null);
+            });
```

Comment:
This can just be:
```C#
Assert.Throws<ArgumentNullException>(() => new UriBuilder((Uri)null));
```
The discard isn't necessary, and it can just be an expression.

Reply:
Fixed these (same for other similar comments)